.github/workflows/docker.yml

```diff
@@ -107,141 +110,9 @@ jobs:
         uses: docker/build-push-action@v2
         with:
           build-args: |
-            SPEC_VERSION=v12
+            SPEC_VERSION=${{ matrix.version }}
```

Comment:
nice! Do they run in parallel?

Reply:
Yeah it creates different jobs for them.